### DISCUSSION TITLE
Story not rendering in version 6.1.15

### DISCUSSION THREAD

**Discussion by ashna99** [storybookjs/storybook#13964] [Help] [answered]
> Currently I am using 6.1.15 version of storybook
> My main.js file:
![image](https://user-images.githubusercontent.com/53310520/107122026-808f9a80-68bb-11eb-9869-52c9773d9d47.png)

> Inside src/components I am having these files
> 1> Button.js
> 2> button.css
> 3>Button.stories.js
> 4>Header.js
> 5>header.css
> 6>Header.stories.js
> But its not rendering stories inside Header.stories.js
> Header.js:
> ![image](https://user-images.githubusercontent.com/53310520/107121783-4a9de680-68ba-11eb-979c-10d6089b3c43.png)
> Header.stories.js:
> ![image](https://user-images.githubusercontent.com/53310520/107121819-76b96780-68ba-11eb-96b3-46551ce6c4e3.png)

Even I have explicitly included Header.stories.js but still the same issue, it's not being rendered. Please see if anyone could help me this as I am really stuck on this issue.

_Originally posted by @ashna99 in https://github.com/storybookjs/storybook/issues/12175#issuecomment-774490831_

**Comment by phated** [comment]
Hey @ashna99! I'm going to convert this to a discussion so it has better visibility for the community to help you out!

**Comment by j3rem1e** [comment, marked as answer]
Fixed here: https://stackoverflow.com/questions/66079281/stories-not-rendering-in-6-1-15-version-of-storybook
